spec/dnsimple/extra_spec.rb

```diff
@@ -26,7 +26,7 @@
   describe ".validate_mandatory_attributes" do
     let(:mandatory_attributes) { %i(name email) }
 
-    it "raises an error if a mandatory attribute is not present " do
+    it "raises an error if a mandatory attribute is not present" do
```

Comment:
This was a rubocop violation that was detected as part of the build that is not related to the other changes in the PR.

Reply:
That was my bad, I've actually fixed that 

https://github.com/dnsimple/dnsimple-ruby/pull/252/commits/4311124c50414e15522fb0a30de7818390745a74

Reply:
👍  Thanks @san983.